src/batch_normalization.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+import {add, sub, mul, div, pow} from './binary.js';
+import {reshape} from './reshape.js';
+import {Tensor} from './tensor.js';
+
+/**
+ * Validate the tensor whether it is a 1-D tensor and its length is equal to `expectedSize`.
+ * @param {Tensor} a
+ * @param {Number} expectedSize
+ * @param {String} name
+ */
+function validateTensor(a, expectedSize, name) {
```

Comment:
I prefer to a more specific name, such as `check1DTesnorWithSize`.

Reply:
Thanks @huningxin. 
Updated it. Please take a look again, thanks.

Reply:
LGTM